resources/batchAccounts.ts

```diff
@@ -0,0 +1,62 @@
+import { Address, BatchRelease, Relationship, UnitError, UnitResponse } from "../types/common"
```

Comment:
I don't see a connection to the unit class 
unit.batchAccounts = new BatchAccount.....

Reply:
added to unit.ts